sdk/storage/storage-file-datalake/src/index.ts

```diff
@@ -5,13 +5,13 @@ export * from "./DataLakeServiceClient";
 export * from "./DataLakeFileSystemClient";
 export * from "./clients";
 export * from "./DataLakeLeaseClient";
-export * from "./AccountSASPermissions";
-export * from "./AccountSASResourceTypes";
-export * from "./AccountSASServices";
-export * from "./AccountSASSignatureValues";
-export * from "./DataLakeSASPermissions";
-export * from "./DataLakeSASSignatureValues";
-export * from "./FileSystemSASPermissions";
+export * from "./sas/AccountSASPermissions";
```

Comment:
Will this be changed in blob, queue and file?

Reply:
Not for now. Can log an issue if we think it's needed.